applications/tests/factories.py

```diff
@@ -36,6 +36,7 @@ class Meta:
 
 class WinterStorageApplicationFactory(BaseApplicationFactory):
     storage_method = WinterStorageMethod.ON_TRAILER
+    municipality = "Helsinki"
```

Comment:
checked this out. actually field `municipality` on the abstract `BaseApplication` model is just a `CharField` and not a `ForeignKey` to `django-munigeo`'s `Municipality` model. I do not know why `municipality = factory.SubFactory(MunicipalityFactory)` works for `BerthApplicationFactory`, that would need a deeper dive into this, but I think we should change it to `municipality = factory.Faker("word")` on `BaseApplicationFactory` to make it consistent with what the model expects.